cranelift/codegen/src/isa/aarch64/lower_inst.rs

```diff
@@ -0,0 +1,306 @@
+//! Lower a single Cranelift instruction into vcode.
```

Comment:
Did you mean to commit this file?

Reply:
It was pulled in after rebasing over more recent work. Thanks for catching that.